test/server/admin/prometheus_stats_test.cc

```diff
@@ -410,8 +433,9 @@ TEST_F(PrometheusStatsFormatterTest, OutputSortedByMetricName) {
   }
 
   Buffer::OwnedImpl response;
-  auto size = PrometheusStatsFormatter::statsAsPrometheus(counters_, gauges_, histograms_, response,
-                                                          false, absl::nullopt);
+  auto size =
+      PrometheusStatsFormatter::statsAsPrometheus(counters_, gauges_, histograms_, response, false,
+                                                  absl::nullopt, Stats::CustomStatNamespacesImpl{});
```

Comment:
suggest instantiating a CustomStatsNamespaceImpl in the test-class rather than constructing it inline. You could also feed it with "wasmcustom." just for consistency but I guess that's not that important for the test.